src/exception_page/version.cr

```diff
@@ -1,3 +1,7 @@
 class ExceptionPage
-  VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
+  macro set_version
+    VERSION = {{ `shards version "#{__DIR__}"`.chomp.stringify.downcase }}
+  end
+
+  set_version
```

Comment:
This is unrelated change. Also, what's the point of creating such a redundant macro?

Reply:
Yeah, I think we can just keep this as is since it still achieves the goal of not having to manage the version.

Reply:
maintains compatibility with [wyhaines/defined.cr](https://github.com/wyhaines/defined.cr#version-checks) version check
so you can do stuff like: https://github.com/wyhaines/opentelemetry-instrumentation.cr/blob/main/src/opentelemetry/instrumentation/frameworks/spider-gazelle.cr#L54

Reply:
basically it makes the VERSION const a little more useful

Reply:
Also, I don't believe we should have a shard specific-hacks in a general-purpose shard like this.

Reply:
@stakach I don't see that how that's achieved. Could you explain, please?

Reply:
I thought that shard just required the constant to be there? It has to be set from within a macro to work? 

/cc. @wyhaines

Reply:
good idea, done!

Reply:
oh, that's wild. I just tried it. I see what you mean @stakach 
https://play.crystal-lang.org/#/r/d285

If you take it out of the begin/end, it doesn't print out.

Reply:
> I thought that shard just required the constant to be there? It has to be set from within a macro to work?

I found the issue when using @wyhaines shard, so slowly updating everything I've previously written

Reply:
ok, I can dig it.